examples/batched-solver/doc/intro.dox

```diff
@@ -1,19 +1,10 @@
-<a name="Intro"></a>
-<h1>Introduction</h1>
-This simple solver example should help you get started with Ginkgo. This example is meant for you to understand
-how Ginkgo works and how you can solve a simple linear system with Ginkgo. We encourage you to play with the code,
-change the parameters and see what is best suited for your purposes.
+<a name="Batched Solver"></a>
+<h1>Using batched solvers</h1>
+This example shows how to use Ginkgo batched solvers with data coming from an
+application. The "application" in this case is just a function in the example
+itself; nevertheless, the steps to be taken are shown.
 
-<h3> About the example </h3>
-Each example has the following sections:
-<ol>
-    <li> <b>Introduction:</b>This gives an overview of the example and mentions
-    any interesting aspects in the example that might help the reader.
-    <li> <b>The commented program:</b> This section is intended for you to
-    understand the details of the example so that you can play with it and understand
-    Ginkgo and its features better.
-    <li> <b>Results:</b> This section shows the results of the code when run. Though the
-    results may not be completely the same, you can expect the behaviour to be similar.
-    <li> <b>The plain program:</b> This is the complete code without any comments to have
-    an complete overview of the code.
-</ol>
+A `batch' here means a set of small linear systems that can be solved
+independently, but each system is too small to use an entire computing device.
+For now, a batch of sparse matrices is required to have a common sparsity
+pattern for all the matrices.
```

Comment:
I'm not sure if this is the best place, but it might be a good idea to mention any pitfalls or limitations, like size/shared memory concerns etc. That might be more relevant in the future than now though so it's fine like this for now.